pebblo/reports/html_to_pdf_generator/report_generator.py

```diff
@@ -63,7 +63,7 @@ def convert_html_to_pdf(data, output_path, template_name, search_path, renderer)
         findings_count = data["reportSummary"].get("findings", 0)
         clientVersion = ""
         versionObj = data.get("clientVersion", None)
-        if versionObj is not None and versionObj.get("version", None) is not None:
+        if versionObj not in [None, {}] and versionObj.get("version", None) is not None:
```

Comment:
Can you also check if just this if statement works if versionObj and versionObj.get("version") ?
Generally if None: will be False and if {} will also be False

Reply:
In both case it will be `False`

Reply:
Updated with affirm cases